.changes/v3.8.0/929-improvements.md

```diff
@@ -0,0 +1,2 @@
+* Resource and data source `vcd_nsxt_network_dhcp` support Isolated networks, different DHCP modes
```

Comment:
I would list those modes here in changelog, so it's instantly clear for the viewers.

Reply:
Added